lib/ecto/adapters/sql/sandbox.ex

```diff
@@ -221,16 +221,16 @@ defmodule Ecto.Adapters.SQL.Sandbox do
           ** (DBConnection.ConnectionError) owner #PID<> timed out
           because it owned the connection for longer than 15000ms
 
-  If you have a long running test, the timeout for the connection ownership may
+  If you have a long running test (or you're debugging with IEx.pry), the timeout for the connection ownership may
   be too short.  You can increase the timeout by setting the
-  `:ownership_timeout` options for your repo config:
+  `:ownership_timeout` options for your repo config (preferably in `config/test.exs`):
```

Comment:
The `config/test.exs` is a phoenix convention that is not enforced by elixir or ecto. It might be confusing here.